README.md

```diff
@@ -1,19 +1,30 @@
 # Monero P2Pool
 
-Decentralized pool for Monero mining.
+Decentralized pool for Monero mining. 
```

Comment:
Unnecessary change?

Reply:
This is an unnecessary change, I wasn't aware of trailing spaces being carried into the changes.